code/modules/robotics/bot/guardbot.dm

```diff
@@ -3544,6 +3544,57 @@
 		check_buddy()
 			return 0
 
+
+	bodyguard/cheer_up
+		name = "cheer_up"
+		task_id = "CHEER_UP"
+		var/global/list/buddy_cheer_up_phrases = list("A magician was walking down the street, then he turned into a grocery store.","Three guys walked into a bar, you’d think at least one of them would have seen it.","You’re having a heart attack, the dispatcher says that help is going to arrive in a heartbeat.","The past, present and future walked into a bar, things got tense.","I asked my French friend if they like videogames, they said Wii.","RIP boiled water, you will be mist.","A guy’s left side was cut off, don’t worry he’s all right now.","What did the sea say to another sea?","You shouldn’t wear glasses while playing football. Why?","Why shouldn’t you get close to trees?","Why are there gates in graveyards?","What do you call an illegally parked frog?","Two silk worms compete in a race, who won?","Whaddya call a cow with no legs?","How does a tree get into his account?","I can't help set the speakers at concerts anymore.")
+		var/global/list/buddy_cheer_up_answers = list("Nothing, they just waved.","Because it’s a contact sport.","Because they’re pretty shady.","Because everyone is dying to get there.","A toad.","None of them, it ended in a tie.","Ground Beef!","It logs in!","Nobody liked my feedback.")
+		var/tmp/initial_seek_complete = 0
+
+		task_act()
+			if (..())
+				return
+
+			if (src.protected && prob(10))
+				var/buddy_cheer_up_chooser = rand(1,16)
+				var/buddy_cheer_up_answer_chooser = (buddy_cheer_up_chooser - 7) //There are more phrases than answers.
+				master.speak(buddy_cheer_up_phrases[buddy_cheer_up_chooser])
+				master.point(src.protected, 1)
+
+				if (buddy_cheer_up_chooser >= 8) //Does the phrase need an answer?
```

Comment:
i'd rather there just be a list of one-off jokes, and then one for the two-parter starters.
so 3 total